documentation/API_documentation/carrier-billing-API-Readiness-Checklist.md

```diff
@@ -1,12 +1,12 @@
 # API Readiness Checklist
 
-Checklist for Carrier Billing v0.3.1 in r1.3
+Checklist for Carrier Billing v0.4.0-rc.1 in r2.1
 
 | Nr | API release assets  | alpha | release-candidate |  initial<br>public | stable<br> public | Status | Comments |
```

Comment:
New column name is referenced within instructions which are copied from the current template:

```suggestion
| Nr | API release assets  | alpha | release-candidate |  initial<br>public | stable<br> public | Status | Reference information |
```